### ISSUE TITLE
error: implicit declaration of function... during installation or building

### ISSUE THREAD

**Issue by dofuuz** [dofuuz/python-soxr#13]
Python-SoXR now supports x86 and ARM processors.
(ppc64le is also supported but it's not being intensively tested.) 

In other platforms, an error like below can occur during pip installation or building.
```
In file included from libsoxr/src/util-simd.c:43,
                 from libsoxr/src/util32s.c:8:
libsoxr/src/pffft.c:186:6: warning: #warning "building with simd disabled !\n"; [-Wcpp]
  186 | #    warning "building with simd disabled !\n";
      |      ^~~~~~~
In file included from libsoxr/src/util32s.c:8:
libsoxr/src/util-simd.c: In function ‘_soxr_ordered_convolve_simd32’:
libsoxr/src/util-simd.c:58:5: error: implicit declaration of function ‘UNINTERLEAVE2’ [-Werror=implicit-function-declaration]
   58 |     UNINTERLEAVE2(a1r, a1i, a1r, a1i);
      |     ^~~~~~~~~~~~~
libsoxr/src/util-simd.c:61:5: error: implicit declaration of function ‘INTERLEAVE2’ [-Werror=implicit-function-declaration]
   61 |     INTERLEAVE2(a1r, a1i, a1r, a1i);
      |     ^~~~~~~~~~~
cc1: some warnings being treated as errors
error: command '/usr/bin/gcc' failed with exit code 1
```

In this case, see [BUILDING.md](https://github.com/dofuuz/python-soxr/blob/main/BUILDING.md#alternative-method-build-using-system-libsoxr) and build Python-SoXR using system libsoxr.

**Comment by dofuuz** [comment]
TODO: Automatically build Python-SoXR using system libsoxr if needed.

**Comment by dofuuz** [comment]
Fixed with v0.5.0a1
